### PR TITLE
Closes: Example 14 talks about minimum luminance in the max luminance section #263

### PR DESCRIPTION
**Pull request by digitaltvguy**
Fixed descriptions of MaxCLL and MaxFall

### CODE REVIEW

**Comment by ProgramMax** [comment]
This pull request seems to be your previous changes + the new change.
I think you meant for the pull request to only include https://github.com/w3c/PNG-spec/pull/267/commits/5cf95d731ee4228ee6cdb16b152c85d95c9a9915

Some existing fixes are causing a conflict in this pull request.

I'll see if the conflicts can be resolved easily so we can squash this pull request and merge.

File: index.html [hunk 1]
```diff
@@ -4000,7 +4000,7 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
               </tr>
 
               <tr>
-                <td>0.005 cd/m<sup>2</sup></td>
+                <td>0.0005 cd/m<sup>2</sup></td>
```
Comment (ProgramMax) [inline index.html:4003]:
```suggestion
                <td>0.005 cd/m<sup>2</sup></td>
```

Oops.
I was able to resolve the conflicts and make this pull request clean.
However, I missed this line.
This should be 0.005, not 0.0005.

**Review by ProgramMax — CHANGES_REQUESTED** [review]
LGTM.
Except while resolving conflicts, I accidentally left in something we don't want to change.

Can you accept the suggested change?
Then I can approve the pull request.

**Comment by digitaltvguy** [comment]
Please recheck my change.  I also updated example in mDCv to show SDR display @ 203cd/m2 instead of 200 (which matches ITU BT.2408-6 pending annex).

**Review by ProgramMax — APPROVED** [review]
LGTM

Although, it looks like the 203 cd/m^2 change didn't get in somehow?
But we can add that in a separate pull request. This pull request is ready to go as-is.

**Pull request merged**